docs/manuals/source/IntroductionAndTutorial/InstallingBareos.rst

```diff
@@ -154,40 +154,77 @@ Debian / Ubuntu
    single: Platform; Debian
    single: Platform; Ubuntu
 
-Bareos :sinceVersion:`15.2.0: requires: jansson` requires the Jansson library package. On Ubuntu is it available in Ubuntu Universe. In Debian, is it included in the main repository.
+Adding an Debian/Ubuntu repository requires multiple steps:
+
+* Storing the Bareos signature key of the repository.
+* Storing the Bareos repository configuration file in :file:`/etc/apt/sources.list.d/`, which references to the repository server a the local key file.
+* If you're a Bareos subscription customer, you also need your login credentials for https://download.bareos.com/ in :file:`/etc/apt/auth.conf.d/`.
+
+To simplify this, the each Debian/Ubuntu repository on https://download.bareos.org/ and https://download.bareos.com/ contain a script named :file:`add_bareos_repository.sh`.
+
+Download the :file:`add_bareos_repository.sh` script
+matching the requested Bareos release
+and the distribution of the target system.
+
+Copy the script onto the target system and execute it with a shell (:command:`sh`) as root (e.g. using :command:`sudo`),
+or manually perform the steps that are documented in the script.
+
+For example the script URL for bareos-21 and Debian 11 is:
+
+* https://download.bareos.org/bareos/release/21/Debian_11/add_bareos_repository.sh
+* or for Bareos subscription customers:
+
+   * https://download.bareos.com/bareos/release/21/Debian_11/add_bareos_repository.sh
+   * .. note::
+
+        To download :file:`add_bareos_repository.sh` you must authenticate against https://download_bareos_com. If this is inconvenient, you can alternatively download :file:`add_bareos_repository_template.sh` and replace ``BAREOS_USERNAME`` and ``BAREOS_PASSWORD`` in there manually.
```

Comment:
Testing this on a blank system is not clear, if you try to download the script from d.b.c without being authenticated, you will get an error 401. So I guess we should be more precise (maybe a link) about `you can download` and add a location.

Reply:
I don't understand what you mean. Can you make a suggestion?